.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "xen"]
 	path = xen
-	url = https://github.com/tklengyel/xen
+	url = https://github.com/xen-project/xen
```

Comment:
Wouldn't it be easier to "upgrade" tklengyel/xen instead of directly going to master? I'm sure in 2 weeks something is going to break...

Reply:
Ah yes, sure.

@tklengyel could you bump `tklengyel/xen` to include `4.15.0-rc1`?

Reply:
No, I don't bump versions to RCs. Also, please don't change the URL.

Reply:
In some previous comments it was asked to enable IPT by default in the `./configure`. We won't be able to merge this until Xen is bumped to 4.15.0-rc1 but I understand we don't want to bump to RC version.

I think IPT should be disabled by default if we want to merge this right now.